tools/tools.go

```diff
@@ -400,13 +364,19 @@ func EnvmanRun(envstorePth,
 		inReader = bytes.NewReader(stdInPayload)
 	}
 
-	cmd := timeoutcmd.New(workDirPth, "envman", args...)
+	name := cmdArgs[0]
+	var args []string
+	if len(cmdArgs) > 1 {
+		args = cmdArgs[1:]
+	}
+
+	cmd := timeoutcmd.New(workDirPth, name, args...)
 	cmd.SetTimeout(timeout)
 	cmd.SetHangTimeout(noOutputTimeout)
 	cmd.SetStandardIO(inReader, outWriter, errWriter)
-	cmd.AppendEnv("PWD=" + workDirPth)
+	cmd.SetEnv(append(envs, "PWD="+workDirPth))
```

Comment:
Why is this better than the existing `AppendEnv()` call?

Reply:
`AppendEnv` is working with a single environment variable + checking if the command has Env set or not is unnecessary because it will be always nil, and the envs we set already contain the os.Environ.

Although we don't need both functions, so I removed `cmd.AppendEnv` in https://github.com/bitrise-io/bitrise/pull/819/commits/73afe9d5f16bb2df5043749d7df0a1488852d61a